### DISCUSSION TITLE
安装脚手架@tarojs/cli (3.6.21)后 执行 taro init projectname 方法报错

### DISCUSSION THREAD

**Discussion by mywei1989** [NervJS/taro#15035] [Q & A] [answered]
# 错误描述
安装脚手架@tarojs/cli (3.6.21)后 执行 taro init projectname 方法报错
<img width="981" alt="image" src="https://github.com/NervJS/taro/assets/2277399/cdcc1b32-ea0b-4a68-a850-ba4c88e2086a">
# 版本信息
<img width="1058" alt="image" src="https://github.com/NervJS/taro/assets/2277399/14b5ee92-7e21-4e98-b112-5bb094cd0d99">

# 操作系统
macox 10.15.7
# 重现方式
npm install -g @tarojs/cli
taro init aaa
# 回退到3.6.20 版本正常

**Comment by xuanzebin** [comment, marked as answer]
3.6.21 引入了 rust 来重构了创建项目的逻辑，这个版本的编译结果只支持 macos 12 以上，存在兼容性问题。

刚刚发布了一个 3.6.22-alpha.5 版本，尝试修复了这个问题，可以升级一下看看还会不会出现创建项目失败的问题～

**Reply by mywei1989** [reply]
@xuanzebin 感谢回复
使用 3.6.22-alpha.5 没有重现该问题

**Reply by xuanzebin** [reply]
好的，这个问题会在这周或者下周跟着 3.6.22 正式版本上线，上线后跟新一下项目中的 Taro 版本即可
